### DISCUSSION TITLE
how to save wakeup on charge settings in the NV memory

### DISCUSSION THREAD

**Discussion by Wahaj6Ahmad** [PiSupply/PiJuice#1022] [Q&A] [answered]
hello hello,

I have a Python script that sets the wakeup on charge settings. But how do I save this in the NV memory? From what I understand, this is what the 'restore' option does in the pijuice gui. I want to achieve the same functionality through the script. any ideas?

Thanks!

**Comment by tvoverbeek** [comment, marked as answer]
Example, setting wakeup threshold to 50%:
```
from pijuice import PiJuice
pj = PiJuice(1,0x14)
pj.power.SetWakeUpOnCharge(50, non_volatile=True)
```

**Reply by Wahaj6Ahmad** [reply]
this worked, thanks!
